zerver/openapi/zulip.yaml

```diff
@@ -279,6 +288,8 @@ paths:
                     }
   /mark_all_as_read:
     post:
+      operationId: mark_all_as_read
```

Comment:
In your commit message, you mentioned "else use most appropriate function name", could you please expand on what you meant by that? Are there any `operationIds` that wouldn't have corresponding functions in `python-zulip-api`?

Reply:
Yes for example I couldn't find fetch_api_key in python-zulip-api.